fedimint-cli/src/client.rs

```diff
@@ -601,6 +609,32 @@ pub async fn handle_command(
             let config = client.get_config_json();
             Ok(serde_json::to_value(config).expect("Client config is serializable"))
         }
+        ClientCmd::ListModules => {
+            #[derive(Serialize)]
+            struct ModuleList<'a> {
+                kind: &'a ModuleKind,
+                id: u16,
+                status: ModuleStatus,
+            }
+
+            let module_list: Vec<ModuleList> = client
+                .get_config()
+                .modules
+                .iter()
+                .map(|(id, ClientModuleConfig { kind, .. })| ModuleList {
+                    kind,
+                    id: *id,
+                    status: if let Some(_value) = client.get_config().modules.get(id) {
+                        ModuleStatus::Active
+                    } else {
+                        ModuleStatus::UnsupportedByClient
+                    },
+                })
+                .collect();
+            return Ok(json!({
+                "List": module_list,
```

Comment:
In no other output we use capital letter keys.

Reply:
changes to `"list"`. Sample output looks like:
```json
{
  "List": [
    {
      "kind": "ln",
      "id": 0,
      "status": "Active"
    },
    {
      "kind": "mint",
      "id": 1,
      "status": "Active"
    },
    {
      "kind": "wallet",
      "id": 2,
      "status": "Active"
    },
    {
      "kind": "meta",
      "id": 3,
      "status": "Active"
    },
    {
      "kind": "unknown",
      "id": 4,
      "status": "UnsupportedByClient"
    }
  ]
}
```